src/components/cc-input-number/cc-input-number.js

```diff
@@ -186,7 +191,7 @@ export class CcInputNumber extends LitElement {
       <div class="meta-input">
         ${controls ? html`
           <button class="btn" @click=${this._onDecrement} ?disabled=${this.disabled || this.readonly || minDisabled}>
-            <img class="btn-img" src=${decrementSvg} alt="">
+            <img class="btn-img" src=${decrementSvg} alt="${i18n('cc-input-number.decrease')}">
```

Comment:
translations:
`decrease`
`décrémenter` (not sure this is a good idea, maybe I should have gone with `augmenter` and `réduire`)